LICENSE.txt

```diff
@@ -1,4 +1,5 @@
 Copyright 2010, Google Inc. All rights reserved.
+Copyright 2024, OpenRefine contributors.
```

Comment:
```suggestion
Copyright 2010, 2024 Google Inc. and OpenRefine contributors. All rights reserved.
```

Reply:
I started contributing in 2010, so I don't think we need to use a separate, later date.

Reply:
Fine by me too!